docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,256 @@
+
+* Making sure it gets marked as belonging to the [Nerves section]
+* Asking clarifying questions to ensure that others will understand the context
+* Offering your own advice or solutions if you can
+* Pinging the core team in the #nerves-dev channel on Slack if it's stale
+
+If you see a question posted in the #nerves channel on Slack, you can help by:
+
+* Asking clarifying questions to ensure that others will understand the context
+* Politely reminding the person that the answer will be more easily found later
+by others if it's posted to the [Elixir Forum] instead
+* Helping to get it posted to the Forum or converted to a GitHub Issue to
+capture or summarize the discussion in Slack about it
+
+## Become a Backer or Sponsor through OpenCollective
+
+The Nerves project has set up an [OpenCollective site] that allows individuals
+and companies to make one-time or recurring financial contributions to cover the
+cost of maintaining the project (e.g. paying for hosted services like AWS and
+CircleCI).
+
+[OpenCollective site]: https://opencollective.com/nerves-project
+
+## Issue and Pull Request triage
+
+In order to keep track of Issues and Pull Requests across all of our
+repositories, we have a [Nerves Radar] project in our GitHub Organization.
+Unfortunately, this isn't visible to the public because GitHub requires that you
+be a member of the Organization to see Organization-level Projects. If you want
+to get involved in this process, get in touch with us on Slack!
+
+We currently don't have this process formalized, so feel free to contribute to
+this document itself!
+
+[Nerves Radar]: https://github.com/orgs/nerves-project/projects/1
+
+## Code and Documentation changes
+
+If you're looking for a way to directly contribute to the code and
+documentation, check our [repositories] for Issues that are labeled with
+`kind:documentation`, `kind:bug`, or `kind:enhancement`. You can use the
+`level:stater`, `level:intermediate`, and `level:advanced` labels to determine
+how difficult we think it will be, whether you're looking for an easy first-time
+contribution or an extra challenge.
+
+If you're working on an Issue, feel free to set yourself as the **Assignee**,
+and don't feel bad about removing yourself if you later decide that you're not
+going to work on it. This helps us keep track of which Issues are currently
+being worked-on, avoiding duplicated effort and Issues that get stale over time.
+
+Working on your first Pull Request? You can learn how from this *free* series,
+[How to Contribute to an Open Source Project on GitHub].
+
+[How to Contribute to an Open Source Project on GitHub]: https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github
+
+### Copyright assignment
+
+**TODO:
+  I mostly made this up based on http://requirejs.org/docs/contributing.html.
+  Review CLA/DCO processes for other OSS projects**
+
+Licensing is very important to open-source software projects, especially for
```

Comment:
My suggestion is that we push off the click-through agreement piece a little longer so that it doesn't hold up the other good things in this PR. How about changing the text to this for now:

Copyright and licensing are very important to open-source software projects. Nerves components fall under several licenses so please review the license for the project to which you are contributing. By contributing, you certify that you have the right to submit the changes under the project's open source license and assign copyright to the Nerves Project maintainers.